core-bundle/src/Resources/contao/library/Contao/Template.php

```diff
@@ -345,6 +346,21 @@ public function route($strName, $arrParams=array())
 		return StringUtil::ampersand($strUrl);
 	}
 
+	/**
+	 * Return a content route relative to the base URL
+	 *
+	 * @param mixed $objContent The content
+	 * @param array $arrParams  The route parameters
+	 *
+	 * @return string The route
+	 */
+	public function contentRoute($objContent, $arrParams=array())
```

Comment:
Is it really necessary to have a separate method? Can we not enhance the existing `route()` method to handle `$objContent`?

Reply:
Changed in a972f8d, though we're no longer generating a route now… I wonder if we should change that in the AbstractController as well? But the parent method in the ControllerTrait has @final on it?

Reply:
Not sure what you mean here. 🙈